### PR TITLE
Add: confirmation before revoking password

### LINKED ISSUE #17: Revoking Passwords: Should we have a confirmation dialog
Currently there is no confirmation dialog for revoking a password.  This can lead to accidental password revoking.  This is especially bad for the revoking all application passwords button.  At a minimum this should have a confirmation dialog.

### PR DESCRIPTION
**Pull request by dinhtungdu**
This PR adds a confirmation box when users click on revoke password buttons. This is the replacement for #23.
Resolve #17.

### CODE REVIEW

**Comment by TimothyBJacobs** [comment]
Thanks for the patch @dinhtungdu!

Could you open this as a Pull Request to the `add/application-passwords` branch in https://github.com/georgestephanis/wordpress-develop/tree/add/application-passwords.

File: class.application-passwords.php [hunk 1]
```diff
@@ -568,11 +568,13 @@ public static function show_user_profile( $user ) {
...
-				'root'      => esc_url_raw( rest_url() ),
-				'namespace' => '2fa/v1',
-				'nonce'     => wp_create_nonce( 'wp_rest' ),
-				'user_id'   => $user->ID,
-				'text'      => array(
+				'root'                 => esc_url_raw( rest_url() ),
+				'namespace'            => '2fa/v1',
+				'nonce'                => wp_create_nonce( 'wp_rest' ),
+				'user_id'              => $user->ID,
+				'revoke_password'      => esc_attr__( 'Are you sure you want to revoke this password? This action cannot be undone.' ),
```
Comment (kasparsd) [inline class.application-passwords.php:575]: Maybe add these strings to the `text` array which already contains one for `no_credentials`?

Comment (dinhtungdu) [reply class.application-passwords.php:575]: Nice catch! Let's do it! Thank you @kasparsd

**Review by kasparsd — APPROVED** [review]
Looks great, thank you @dinhtungdu!

**Comment by dinhtungdu** [comment]
@TimothyBJacobs I made a PR to [add/application-passwords](https://github.com/georgestephanis/wordpress-develop/tree/add/application-passwords): https://github.com/georgestephanis/wordpress-develop/pull/1

**Pull request merged**